docs/standard/base-types/string-comparison-net-5-plus.md

```diff
@@ -29,8 +29,21 @@ Console.WriteLine($"{greeting.IndexOf("\0")}");
 // '0' when running on .NET 5 or later (Windows)
 // '0' when running on .NET Core 2.x - 3.x or .NET 5 (non-Windows)
 // '3' when running on .NET Core 2.x or .NET 5+ (in invariant mode)
+
+string s = "Hello\r\nworld!";
+int idx = s.IndexOf("\n");
+Console.WriteLine(idx);
+
+// The snippet prints:
+//
+// '6' when running on .NET Core 3.1 (Windows version before Windows 10 May 2019 Update)
```

Comment:
>Windows version before Windows 10 May 2019 Update [](http://example.com/codeflow?start=38&length=49)

please remove the part "version before Windows 10 May 2019 Update". 3.1 will always give this result on any version of Windows.